src/app/main/component/layout/components/header/header.component.ts

```diff
@@ -222,4 +223,8 @@ export class HeaderComponent implements OnInit, OnDestroy {
   public toggleScroll(): void {
     this.toggleBurgerMenu ? document.body.classList.add('modal-open') : document.body.classList.remove('modal-open');
   }
+
+  setUbsRegistration(value: string): void {
+    this.localStorageService.setUbsRegistration(value);
+  }
```

Comment:
Why this method should take string if you always pass 'true' or 'false' to it. I suggest changing it to boolean and pass bool  (Of course, if it only works with true or false)